src/averageAges.js

```diff
@@ -55,7 +77,20 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const lol = people
```

Comment:
What is lol?)
Try to give clearly name to your variables
```suggestion
  const lol = people
```